iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/iot/IoTConsensusServerImpl.java

```diff
@@ -909,6 +916,7 @@ private TSStatus cacheAndInsertLatestNode(DeserializedBatchIndexedConsensusReque
         for (IConsensusRequest insertNode : request.getInsertNodes()) {
           subStatus.add(stateMachine.write(insertNode));
         }
+        ioTConsensusServerMetrics.recordApplyCost(System.nanoTime() - sortTime);
```

Comment:
Should we count the time for each IConsensusRequest, otherwise it's not the time of applying the state machine for a single request

Reply:
Yes. Now it's the time of one Batch. We should divide it to calculate for every single request.